src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/EnumeratingWhitelist.java

```diff
@@ -223,6 +223,13 @@ private boolean exists(Class<?> c, boolean start) throws Exception {
         @Override public String toString() {
             return "staticMethod " + signaturePart();
         }
+        @Override boolean exists() throws Exception {
+            try {
+                return Modifier.isStatic(type(receiverType).getDeclaredMethod(method, types(argumentTypes)).getModifiers());
```

Comment:
Otherwise `StaticWhitelistTest.sanity` would fail, claiming `Hudson.getInstance` was an override. (`exists` is used only from tests so this is not a user-visible change.)